### DISCUSSION TITLE
Mantle is not even working in my level

### DISCUSSION THREAD

**Discussion by Norsedevstudios** [Sixze/ALS-Refactored#143] [Q&A]
Not sure if there is an issue or not, Same download as my buddy from github same setup but his works and mine does not?  it worked one time but when I played again it didnt.  Debug shows it as a possibility to mantle but character does not.

**Comment by Sixze** [comment]
I can't say anything without details. Mantling should by default work with the `WorldStatic`, `WorldDynamic` and `Destructible` object types.

**Comment by Midlex** [comment]
having the same issue had ragdoll making my editor crash but mantling worked, now after i made a new empty project and reinstalled the plugin, ragdoll works but mantling does not, when i try to mantle i get a debugbreak popup on vs.
